.github/workflows/ci.yml

```
@@ -14,12 +14,18 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
+      - run: echo "node_version=$(cat .nvmrc)" >> $GITHUB_ENV
       - uses: actions/setup-node@v3
         with:
-          node-version-file: ".nvmrc"
+          node-version: ${{ env.node_version }}
           cache: "npm"
       - run: npm ci --ignore-scripts
       - run: npm test
+      # Upload coverage for sonarcube (only matching OS and one node version required)
+      - uses: actions/upload-artifact@v3
+        with:
+          name: code-coverage-ubuntu-latest-${{ env.node_version }}
```

```suggestion
          name: code-coverage-report
```

The node version doesn't matter here.